http/swagger.yml

```diff
@@ -6088,6 +6088,9 @@ components:
         id:
           readOnly: true
           type: string
+        oauthID:
+          readOnly: true
+          type: string
```

Comment:
Looks like the oauthID is added to the session, which is part of authorization, not really part of a user.
I suggest maybe we could make a Me schema, which includes user response and authorization response.

Reply:
Updated the User interface to include the oauthID instead of the session